.eslintrc

```diff
@@ -9,7 +9,7 @@ parserOptions:
     globalReturn: false
     impliedStrict: true
     jsx: true
-    experimentalObjectRestSpread: false
+    experimentalObjectRestSpread: true
```

Comment:
Does this need an update in https://github.com/cfpb/development/blob/master/.eslintrc#L12 (and CF and generator-cf)? This is to avoid usage of `Object.assign`?

Reply:
It's my understanding that object rest spread has reached stage 4 per: https://github.com/tc39/proposal-object-rest-spread.  I think its easier to read and less code.

Reply:
Works for me! We should adjust the other eslints to match I'd say, so https://github.com/cfpb/development/blob/master/.eslintrc#L12 and https://github.com/cfpb/capital-framework/blob/canary/.eslintrc#L12